cg/services/orders/order_lims_service/order_lims_service.py

```diff
@@ -14,23 +14,35 @@ def __init__(self, lims_api: LimsAPI):
         self.lims_api = lims_api
 
     @staticmethod
-    def _build_lims_sample(customer: str, samples: list[OrderInSample]) -> list[LimsSample]:
+    def _build_lims_sample(
+        customer: str, samples: list[Sample], workflow: Workflow, delivery_type: DataDelivery
+    ) -> list[LimsSample]:
         """Convert order input to lims interface input."""
         samples_lims = []
         for sample in samples:
-            dict_sample = sample.__dict__
+            dict_sample = sample.model_dump()
             LOG.debug(f"{sample.name}: prepare LIMS input")
             dict_sample["customer"] = customer
+            dict_sample["data_analysis"] = workflow
+            dict_sample["data_delivery"] = delivery_type
```

Comment:
A bit uncertain how we should get these attributes here in the smoothest way - LIMS is expecting case attributes on its samples